ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/CatalogController.java

```diff
@@ -176,8 +178,8 @@ public void getCatalogTypeList(Context ctx) {
   }
 
   /** Convert server auth config to metaAuthConfig */
-  private Map<String, String> authConvertFromServerToMeta(
-      Map<String, String> serverAuthConfig, CatalogMeta oldCatalogMeta) {
+  private void authConvertFromServerToMeta(
```

Comment:
Maybe change the method name to `fillAuthConfigs` or `fillAuthConfigs2CatalogMeta` as you have changed the behavior and return type of this method.

Reply:
I rename it to `fillAuthConfigs2CatalogMeta`